packages/react-table/src/components/Table/examples/ComposableTable.md

```diff
@@ -0,0 +1,1290 @@
+---
+id: TableComposable
+cssPrefix: pf-c-table
+section: components
+propComponents:
+  [
+    'BaseTable',
+    'BaseTableCaption',
+    'BaseTableHead',
+    'BaseTableBody',
+    'BaseTableHeaderRow',
+    'BaseTableBodyRow',
+    'BaseHeaderCell',
+    'BaseBodyCell',
+  ]
+ouia: true
+beta: true
+---
+
+Note: Table lives in its own package at [@patternfly/react-table](https://www.npmjs.com/package/@patternfly/react-table)!
+
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
```

Comment:
Do you not get these imports for free as well from the theme?

Reply:
doesn't seem like it, if I remove them the component is not styled correctly